AWS VPC peering article

dewan-ahmed opened the pull request:
# What changed, and why it matters

This is a migration task which resolves https://github.com/aiven/devportal/issues/1106

Original help article: https://help.aiven.io/en/articles/778836 (broken down into multiple articles)

ftisiot commented on docs/platform/howto/manage-vpc-peering.rst:
```diff
@@ -34,6 +34,7 @@ Cloud-specific VPC peering instructions
 -----------------------------------------------------
 
 - :ref:`Set up VPC peering on Google Cloud Platform (GCP) <vpc-peering-gcp>`.
+- :ref:`Set up VPC peering on Amazon Web Services (AWS) <vpc-peering-aws>`.
```
Should this be a `:doc:` and not a `:ref:` ?

ftisiot commented on docs/platform/howto/vpc-peering-aws.rst:
```diff
@@ -0,0 +1,21 @@
+.. _vpc-peering-aws:
```
I believe once we move the above reference to a doc, this becomes useless

ftisiot reviewed (dismissed):
A couple of modifications needed for consistency

dewan-ahmed commented:
Hi @ftisiot I think I addressed the review comments already but GitHub is still showing "1 change requested". Pls let me know in case I missed anything. Thanks.
